src/index.ts

```diff
@@ -1,8 +1,14 @@
 /* eslint-disable @typescript-eslint/no-explicit-any */
 import type MarkdownIt from "markdown-it/lib"
+import { escapeHtml } from "markdown-it/lib/common/utils"
+import Renderer from "markdown-it/lib/renderer"
```

Comment:
```suggestion
import type Renderer from "markdown-it/lib/renderer"
```

this is just for typing no?